src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/UI/DontShowAgainMessageBox.xaml

```diff
@@ -0,0 +1,60 @@
+﻿<ui:DialogWindow
```

Comment:
This handles all the `EnableModeless` stuff VS requires, right?

Reply:
Yes. Handled in the base class in from the IDE